src/topics.ts

```diff
@@ -33,3 +33,15 @@ export function compileTopics(
 function hexToBytes(input: string): Uint8Array {
   return Buffer.from(input.substring(2), 'hex');
 }
+
+export function toStringArray(input: string): Array<string>{
+  return input
+  .split(',')
+  .map(
+    (t: string) => 
+    t.replace("{\"\\\\", '0')
+    .replace("\"\\\\", "0")
+    .replace("\"}", "")
+    .replace("\"", "")
+  );
+}
```

Comment:
If run through the linter, it should complain about the missing trailing newline